Source/WebCore/Modules/webaudio/ScriptProcessorNode.h

```diff
@@ -85,7 +85,7 @@ class ScriptProcessorNode final : public AudioNode, public ActiveDOMObject {
     // Double buffering.
     static constexpr unsigned bufferCount = 2;
     unsigned bufferIndex() const { return m_bufferIndex; }
-    void swapBuffers() { m_bufferIndex = (m_bufferIndex + 1) % bufferCount; }
+    void swapBuffers() { m_bufferIndex ^= 1; }
```

Comment:
^= 1 alternates between 0 and 1.

Reply:
0 ^ 1 is 1

1 ^ 1 is 0

Reply:
Sorry, you're right of course. I guess my brain was off went I wrote this.

This is OK so long as bufferCount remains 2. But the code is otherwise designed so that it still works if bufferCount is increased. It won't work if bufferCount was increased to 3, for example. So we still shouldn't change this.

Also, it's not as clear as the original code, so I would recommend against making this change regardless.